### ISSUE TITLE
Compact syntax for settings.xml

### ISSUE THREAD

**Issue by o-sdn-o** [netxs-group/vtm#389]
The following declarations should be equivalent
```xml
<block>
    <item>
        <param=value/>
    </item>
</block>
```

```xml
<block/item/param=value/>
```

It turns out that the following declarations are also equivalent
```xml
<block>
    <param>
        <nested=value>
            <block1=value1/>
            <block2>
                <attr=value2/>
            </block2>
        </nested>
    </param>
</block>
```
```xml
<block/param/nested=value block1=value1 block2/attr=value2 />
```

**Comment by o-sdn-o** [comment]
It turns out that the following declarations are also equivalent
```xml
<block>
    <item>
        <param=value1>
            <nested=value2>
                <block1=value3/>
                <block2=value4/>        
            </nested>
        </param>
    </item>
</block>
```
```xml
<block/item/param=value1 nested=valule2 nested/block1=valule3 nested/block1=valule4 />
```

**Comment by o-sdn-o** [comment]
This syntax modification does more harm than good. It's easier to generate a configuration with a script than manually write it out with a cool syntax.

Closed in favor of #393.
